### DISCUSSION TITLE
American sort and Radix sort

### DISCUSSION THREAD

**Discussion by DurjoyAcharya** [DurjoyAcharya/Algorithm#5] [Q&A] [answered]
What is the difference between american sort and radix sort?

**Comment by darkdev3v** [comment, marked as answer]
Radix sort: Time: O(nk), Space: O(n+k), n is the number of keys, k is the maximum number of digits a number (value) can have.

American flag sort: Time: O(n*k/d), Space: O(k), n is the number of digits and k is the average bucket size.
